src/components/Header.tsx

```diff
@@ -1,11 +1,38 @@
-import { Link } from "react-router-dom"
+import  { useState } from 'react';
+import { Link } from 'react-router-dom';
 
+export default function Header() {
+    const [isDropdownOpen, setIsDropdownOpen] = useState(false);
 
-export default function Header(){
+    const toggleDropdown = () => {
+        setIsDropdownOpen(!isDropdownOpen);
+    };
 
-    return(
+    return (
         <div className='navbar'>
-        <Link to ={`/doclist`}><p>Doc</p></Link>
+<<<<<<< HEAD
```

Comment:
Just like the other files, it looks like there are git conflicts on this page. Please resolve all conflicts